github.js

```diff
@@ -71,7 +74,17 @@ var GithubLocation = function(options, ui) {
     throw 'Git not installed. You can install git from `http://git-scm.com/downloads`.';
   }
 
-  this.strictSSL = 'strictSSL' in options ? options.strictSSL : true;
+  this.defaultOptions = {
+    strictSSL: 'strictSSL' in options ? options.strictSSL : true
+  };
+
+  for (var i = 0, keys = ["ca", "cert", "key"]; i < keys.length; i++) {
```

Comment:
Should we support `passphrase` as well here? Are all these options mandatory, or should some be optional?